ios-target/View/Chat/ChatListViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ChatConversationViewController.swift
+//  ios-target
+//
+//  Created by sol manini on 7/16/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import UIKit
+
+class ChatListViewController: UIViewController {
+  
+  @IBOutlet weak var chatsTableView: UITableView!
+  
+  var viewModel = ChatListViewModel()
+  
+  var unwindChatSegue = "unwindChatSegue"
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    viewModel.delegate = self
+    viewModel.loadConversations()
+  }
+  
+  func setupView() {
+    chatsTableView.register(UINib(nibName: ChatListTableViewCell.reuseIdentifier,
+                                  bundle: nil), forCellReuseIdentifier: ChatListTableViewCell.reuseIdentifier)
+    chatsTableView.dataSource = self
+    chatsTableView.delegate = self
+  }
+}
+
+extension ChatListViewController: ChatListViewModelDelegate {
+  func didLoadMatches() {
+    setupView()
```

Comment:
I think you need to call this on `viewDidLoad` and reload tableView data on `didUpdateState`